src/swarm-lang/Swarm/Language/Syntax/Pretty.hs

```diff
@@ -70,6 +70,7 @@ instance PrettyPrec (Term' ty) where
     TAntiInt v -> "$int:" <> pretty v
     TText s -> fromString (ushow s)
     TAntiText v -> "$str:" <> pretty v
+    TAntiSyn v -> "`" <> pretty v
```

Comment:
Why the tick? I would have expected `$var:`.

Wouldn't this be problematic if we added infix call like Haskell has with ticks on both sides of the identifier?

Reply:
I don't know, I just thought it looked cool.  I can change it to `$syn:` to be consistent with the other ones.